recipes/msgpack-cxx/all/conanfile.py

```diff
@@ -23,18 +32,14 @@ def _source_subfolder(self):
     def _build_subfolder(self):
         return "build_subfolder"
 
-    def requirements(self):
-        self.requires("boost/1.78.0")
-
-    def validate(self):
-        if self.options["boost"].header_only == True:
-            raise ConanInvalidConfiguration("{} requires boost with header_only = False".format(self.name))
+    def configure_options(self):
+        # No boost was added in 4.1.0
+        if tools.Version(self.version):
```

Comment:
Need to compare some version here. Only converted string  to Version.

Reply:
fixed